docs/sphinx/source/changelog/v2.0.0.rst

```diff
@@ -4,14 +4,18 @@ v2.0.0 (MONTH DAY, YEAR)
 
 API Changes
 -----------
+* The behavior of :py:func:`~rdtools.normalization.normalize_with_pvwatts`
+  and :py:func:`~rdtools.normalization.normalize_with_sapm` have changed to
+  expect input energy in the moving average form calculated by
+  :py:func:`~rdtools.normalization.energy_from_power` (:pull:`105`, :pull:`108`).  
```

Comment:
I would characterize it as a trapezoidal rule integration rather than a moving average, but that may be moot because I suggest revising to:

>The calculations internal to normalize_with_pvwatts and normalize with sapm have changed. Generally, when working with raw power data it should be converted to right-labeled energy with energy_from_power before being used with the normalization functions.

The PR links will allow folks to dig into the details if they need to.

Reply:
The moving average bit was quoting this: https://github.com/NREL/rdtools/blob/development/rdtools/normalization.py#L502

But agreed that your version is better anyway.